src/components/forms/fragments/exchangeApplication/ExchangeApplicationContainer.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Query } from 'react-apollo';
+
+import { BerthSwitchReasonsQuery } from '../../../../utils/__generated__/BerthSwitchReasonsQuery';
+import { BERTH_SWITCH_REASONS_QUERY } from '../../../../utils/graphql';
+import ExchangeApplication, { ExchangeApplicationProps } from './ExchangeApplication';
+
+type Props = Pick<ExchangeApplicationProps, Exclude<keyof ExchangeApplicationProps, 'reasons'>>;
```

Comment:
Lately, Typescript has dropped `Omit` in favor of `Exclude`
> We did not include the Omit<T, K> type because it is trivially written as Pick<T, Exclude<keyof T, K>>.

Reply:
What I meant is that creating a custom type helper would be more readable. Anyways TypeScript 3.5 seems to have Omit so if this project is upgraded to that then I would recommend to use built-in Omit, maybe add a TODO-comment here for that? https://devblogs.microsoft.com/typescript/announcing-typescript-3-5/#the-omit-helper-type

Reply:
Nice to know that `Omit` will be included in v3.5.